LaFrish.txt

```diff
@@ -0,0 +1,5 @@
+Will I ever learn
+Procrastinated again
+Sleepy girl laments
+
```

Comment:
Funny story about the sleepy girl line is that a friend of mine actually was featuring me on his blog as a recurring character named sleepy girl. Bc of my insomnia I would send random emails or post about not sleeping abt not sleeping. He thought it was funny to feature some of these amusements on his blog and everyone loved it... I had no idea he was doing this but his followers and other friends loved it. There is always a story to everything in my life :)